live_data/websocket_subscriber/test_scripts/ws_server.py

```diff
@@ -0,0 +1,46 @@
+import asyncio
```

Comment:
If you want to leave it for testing it's ok, however the server I am currently implementing is quite different from this implementation (mainly because there is no need to broadcast a message, and also because I have to wait 15 mins before every transmission)

Reply:
Will leave it for future to instantly test it